maxprocs/maxprocs.go

```diff
@@ -66,7 +59,23 @@ func Logger(printf func(string, ...interface{})) Option {
 func Min(n int) Option {
 	return optionFunc(func(cfg *config) {
 		if n >= 1 {
-			cfg.minGOMAXPROCS = n
+			cfg.MinValue = n
+		}
+	})
+}
+
+// ReservedCPUQuota specifies a CPU quota amount to consider reserved for use
+// outside of the Go runtime; e.g. by a cgo extension, or another process
+// within the container.
+//
+// GOMAXPROCS will be set to the remaining (rounded down) quota, clamped to the
```

Comment:
Is "rounded down" still the intention? This part of the comment contradicts the use of `Ceil` in `CPUQuotaToGOMAXPROCS` (commented there).

Reply:
Yes it is, this doc comment is out of line with the code reality above, good catch.